src/commands/database/mysql/user/create.tsx

```diff
@@ -0,0 +1,112 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import { Value } from "../../../../rendering/react/components/Value.js";
+import type { MittwaldAPIV2Client } from "@mittwald/api-client";
+import { mysqlUserFlagDefinitions } from "../../../../lib/resources/database/mysql/user/flags.js";
+import { Flags } from "@oclif/core";
+
+type Result = {
+  mysqlUserId: string;
+};
+
+type MyQSLUserCreationData = Parameters<
+  MittwaldAPIV2Client["database"]["createMysqlUser"]
+>[0]["data"];
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static description = "Create a new MySQL user";
+  static flags = {
+    ...processFlags,
+    "database-id": mysqlUserFlagDefinitions["database-id"]({ required: true }),
+    "access-level": mysqlUserFlagDefinitions["access-level"]({
+      required: true,
+    }),
+    description: mysqlUserFlagDefinitions.description({ required: true }),
+    password: mysqlUserFlagDefinitions.password({ required: true }),
+    "access-ip-mask": mysqlUserFlagDefinitions["access-ip-mask"]({
+      dependsOn: ["enable-external-access"]
```

Comment:
Thanks for nothing, eslint 😖 

```suggestion
      dependsOn: ["enable-external-access"],
```